pdm/test/action.yml

```diff
@@ -58,7 +58,7 @@ runs:
       shell: bash
 
     - name: Start and wait Docker Compose
-      if: inputs.docker-compose == 'true' and report-only == null
+      if: inputs.docker-compose == 'true' && report-only == null
```

Comment:
This should be `inputs.report-only`

Reply:
Yep, I saw that too late, sorry
Fixed in #267